index.html

```diff
@@ -0,0 +1,173 @@
+                <li class="nav-item">
+                    <a href="#sponsor" class="nav-link">Sponsor</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#news" class="nav-link">News</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#campaign" class="nav-link">CB Campaign</a>
+                </li>
+            </ul>
+            <!-- Hamburger -->
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </nav>
+    </header>
+
+    <main>
+        <!-- Front page -->
+        <section class="event-head" id="home">
+            <div class="event-line">
+                <h2 class="event-greeting">"Hello! Sharing world"</h2>
+                <h1 class="rainbow-text">PROUD 2 PAY MARKETING CAMPAIGN EVENT 2023</h1>
+                <p>“We have technology, finally, that for the first time in human history <span id="content"></span> allows people to really maintain rich connections with much larger numbers of people.”</p>
+            </div>
+            <div class="event-date">
+                <h2>2023.08.23(WED) ~ 25(FRI)</h2>
+                <p>@Zoom Meet</p>
+            </div>
+        </section>
+
+        <!-- Feature -->
+        <section class="program" id="program">
+            <h2 class="program-text">Main Program</h2>
+            <span class="highlight"></span>
+            <div class="program-list">
+                <!-- Card 1 -->
+                <div class="program-card">
+                    <img src="imgs/distribution-icon.png" alt="Marketplace">
+                    <h3 class="program-title">Distribution</h3>
+                    <p>“The best supply chain is one that has no beginning and no end.”</p>
+                </div>
+                <!-- Card 2 -->
+                <div class="program-card">
+                    <img src="imgs/digital-marketing.png" alt="Affiliate">
+                    <h3 class="program-title">Marketing</h3>
+                    <p>“Business has only two functions — marketing and innovation.”</p>
+                </div>
+                <!-- Card 3 -->
+                <div class="program-card">
+                    <img src="imgs/promotion.png" alt="blog">
+                    <h3 class="program-title">Promotion</h3>
+                    <p>“It's not what you sell that matters as much as how you sell it!”</p>
+                </div>
+                <!-- Card 4 -->
+                <div class="program-card">
+                    <img src="imgs/educate.png" alt="Education">
+                    <h3 class="program-title">Education</h3>
+                    <p>“Education is one thing no one can take away from you.”</p>
+                </div>
+            </div>
+            <a href="#" class="program-text" type="button"> SEE THE WHOLE PROGRAM</a>
+        </section>
```

Comment:
Please kindly align the content of this section at the center to match the template

Your design
![image](https://user-images.githubusercontent.com/78251927/212147904-da383228-437c-4fd6-94fc-cb0ade8949a8.png)

Template
![image](https://user-images.githubusercontent.com/78251927/212147725-d3959ea6-8e0a-4b3c-95b4-42dcd9ea26fa.png)